### ISSUE TITLE
Please remove pdpPinkie avatar

### ISSUE THREAD

**Issue by MiraWaNeko** [JTBrinkmann/plug_p0ne#36]
Please remove the pdpPinkie avatar as this was never released by the artist to be used by any other scripts than the original Plug.Pony or later scripts given permission by me.

**Comment by JTBrinkmann** [comment]
I'm investigating this

**Comment by JTBrinkmann** [comment]
It seems to me that your claim is simply not true. The avatar in question was created for the plug.dj users, specifically in the [friendshipismagic community](https://plug.dj/friendshipismagic); there doesn't seem to be any indicator that they were made exclusively for any scripts that you give permission to, nor would that make much sense to begin with. In fact, you are known for sabotaging the one community that the avatar(s) was(/were) made for, and you are known for trying to restrict people's access to them. This is neither in the interest of the artist, nor of the community. Additionally, I remember having seen you several times offering to share the Plug.Pony avatar files you have/had access to, making your request even more obscure.

I am closing this issue until you have more helpful input on the subject.

Personally, I suggest you to stop claiming control over other people's work, to stop using unjustified made-up claims and to improve your skills in writing issues on issue trackers, as you clearly seem to be involved in other projects which would also benefit from you writing proper reports.

**Comment by AmpDragoness** [comment]
Hello. I am he original artist of the Pinkie pie avatar plus a few others I did for the Plug.Pony project. I have not given any permission to anyone else to use them and would appreciate you taking them down. 
http://imgur.com/a/SrUhG
if you would like to speak to me further about this please contact me via twitter @ampDragoness
Thank you for understanding.

**Comment by JTBrinkmann** [comment]
thank you for reaching out, @AmpDragoness! I take this issue **very** seriously and I **do** respect your rights.

The good news is: while I was unable to contact you to resolve this issue, I precautiously removed your avatars **in April 2017 already** and haven't put them back up since.

However, I'd like to ask you to **reconsider giving me permission**. Read below why.


# Full Story
> **tl;dr** me and fimplug staff assumed your avatars were "for the fimplug community", not "for Plug.Pony" in particular; plug_p0ne was crowned the "official Plug.Pony successor". Attempts were made to contact you to explicitly ask for permission.

Background story: the community friendshipismagic (aka. "fimplug") is one of the oldest, still active communities on plug.dj. The addon Plug.Pony which was made specifically for fimplug added a lot of ✨ magic ✨  to the site and was loved by many, including me. It had the ability to add custom (pony) avatars, which I was told were kept in a shared folder, accessible by (some) moderators and artists.
Plug.Pony's artwork was contributed **by the community**. It is my impression that Plug.Pony (specifically its artwork) is a community effort and is made **for the community**, not the addon in specific.
Sadly, development of Plug.Pony has stopped, moderators were deprived of their access to said shared folder, the server went down, and major updates to plug.dj completely broke the Plug.Pony.

How p0 became Plug.Pony's successor: Upset with that situation, I decided to spend a lot of my free time and energy to work voluntarily and unpaid on a successor of Plug.Pony. I talked a lot to the community's staff, including the (co-)hosts Sgt Chrysalis ("DJ Chrysalis") and Unnoen ("ʊηᴋηæωη"), who supported my efforts and agreed that I may/should take over the responsibility of developing the community's addon, Plug.Pony. Because TAT and Colgate abandoned both fimplug and Plug.Pony, neither of us assumed there would be conflicts with this decision. Just to be extra safe though, I decided to not use any previous Plug.Pony code and to adopt a new name (plug_p0ne aka. p0). As it was and is considered (by fimplug) the "official Plug.Pony successor", it was thought to be appropriate to continue using the artwork that was made **for** the fimplug community to provide it **to** the fimplug community.

How avatars were recovered: Sadly *all* of the original contributors already left fimplug as well and most even changed nicknames and/or deleted their old online accounts/blogs. Except for EvoStorm, I personally didn't know the artists (partly because timezone differences kept us online at different times). To make matters worse, Plug.Pony itself didn't credit the artists and lacked any (still available) list of avatars or info about their licenses. I spent many, many weeks trying to find all artists to regain the lost custom avatars and to ensure explicit permission is given to use them. I was actively in contact with both fimplug staff, old users and even TAT, but couldn't compile a complete list of previous avatars and (still working) contact information of the artists. In fimplug chat, TAT ("Chikachi") repeatedly promised his help, but never actually followed through.

Regarding pdpPinkie: IIRC, I got the pdpPinkie files from either EvoStorm or Zeezal, who still had them "lying around". I added them to p0 in good faith that it is **the artist's original intention** for their work to be available to fimpluggers (especially given that they did a great job on them! 👌)  Luckily [plugpony.tumblr.com](https://plugpony.tumblr.com/post/29921833162/welcome-to-amp) mentions that it was made by "Amp" with a link to [DiamondDuskPony.tumblr.com](http://diamondduskpony.tumblr.com/) (which is since down). But still, after spending a good amount of time googling all names in multiple variations and asking around, I simply couldn't find any still active account.

After this issue was opened (regarding pdpPinkie), I again tried my best to research ([example screenshot](https://i.imgur.com/etHkusv.png)), yet I couldn't get in touch. I really tried. **I am deeply sorry** if you considered your art to have been "stolen", and I'm doing my best to make up for it. 🦄

**Comment by JTBrinkmann** [comment]
### How to continue
As mentioned before, I have already removed your avatars. I am working on an improved version of p0 (to properly show credits for avatars) and of its website (to make it easier for artists to manage their submitted avatars). How does that sound to you, @AmpDragoness? I'm sure fimpluggers would be happy to get back the pony avatars they love, and I'd appreciate it if you would kindly submit yours to p0.

Btw, which other ones did you make?
